src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -297,6 +314,10 @@ TaskActivity INameVersionObjectManager<TaskActivity>.GetObject(string name, stri
                     context.InstanceId,
                     context.IsReplaying);
             }
+
+ 
+                await context.RunDeferredTasks();

```

Comment:
Please only use 4 spaces for indenting. Also, please remove the extra newlines above and below.

Reply:
Thank you! We will improve the presentation style here.